test/suite_test.go

```diff
@@ -262,7 +267,6 @@ func TestSuite(t *testing.T) {
 		if testImage == "" {
 			testImage = "localhost/" + config.OperatorName + ":latest"
 		}
-		selinuxdImage = "quay.io/security-profiles-operator/selinuxd-fedora"
```

Comment:
What were the main differences between using this image and the one we're setting on line 200?

Reply:
I suspect this was used in the fedora VM?